x-pack/plugin/async-search/src/internalClusterTest/java/org/elasticsearch/xpack/search/BlockingQueryBuilder.java

```diff
@@ -127,6 +128,11 @@ public String getWriteableName() {
         return NAME;
     }
 
+    @Override
+    public Version getMinimalSupportedVersion() {
+        return Version.V_EMPTY;
```

Comment:
not too sure when this was introduced. I am thinking during the 7.x series.

Reply:
This query builder looks like something used only in tests though, I didn't give those a thorough check.